docs/pages/more/glossary-of-terms.mdx

```diff
@@ -137,9 +137,9 @@ The command-line tool for working with Expo. This term now refers to the [Local
 
 The former name for the [Expo Go](#expo-go) app.
 
-### Expo Config
+### App config
 
-A file named `app.json`, `app.config.json`, `app.config.js`, or `app.config.ts` in the root of a project directory. For more information, see [Expo Config configuration](/workflow/configuration/).
+A file named **app.json**, **app.config.json**, **app.config.js**, or `app.config.ts` in the root of a project directory. For more information, see [app config configuration](/workflow/configuration/).
 
 This file is used for the following purposes:
```

Comment:
This section need to be moved up after term rename.

Reply:
Done ✅